api/apps/api/src/modules/projects/projects-crud.service.ts

```diff
@@ -78,6 +79,8 @@ export class ProjectsCrudService extends AppBaseService<
         'scenarios',
         'createdAt',
         'lastModifiedAt',
+        'planningAreaId',
+        'planningAreaName',
```

Comment:
This will not work, because `_processBaseFilter()` will try to add a SQL `AND WHERE` but these are not db columns

Reply:
but it shouldn't be a problem to drop these two elements from this array as we'd not need to filter projects by area id or name

Reply:
@hotzevzl 
![image](https://user-images.githubusercontent.com/3466388/121009511-8b3c3c80-c794-11eb-9115-9d5f3f18c32f.png)

without those, fields are not available in response. What do you recommend?

Reply:
(feel free to play with it `project/project-get.e2e-spec.ts:` )

Reply:
sorry, total oversight on my side - I read this in the wrong context because of the way it was presented in the PR diff. I was convinced I was looking at the array of prop names against which to filter. Ignore 😬